assets/js/romo/base.js

```diff
@@ -881,6 +860,27 @@ Romo.prototype._elemsWrapMap = {
   'td':       [3, "<table><tbody><tr>", "</tr></tbody></table>"]
 };
 
+Romo.prototype._elemsInitComponents = {};
+
+Romo.prototype._elemsInitTrigger = function(onElems) {
+  for (var selector in this._elemsInitComponents) {
+    this._elemsInitFind(onElems, selector).forEach(Romo.proxy(function(initElem) {
+      new this._elemsInitComponents[selector](initElem);
+    }, this));
+  }
+}
+
+Romo.prototype._elemsInitFind = function(onElems, selector) {
+  var elems = Romo.find(onElems, selector);
+  onElems.forEach(function(onElem) {
```

Comment:
@kellyredding - You could do use `elems.concat` and `filter` to maybe shorten this? Something like:

```javascript
elems.concat(onElems.filter(function(onElem) {
  return Romo.is(onElem, selector);
});
```

I don't think it's any more efficient but its less lines.

Reply:
@jcredding thanks for this suggestion.  I did what you suggested except I started with the onElems that matched and concatted the child elems that matched.  I wanted the parent elems that matched to be listed before the child elems so that the parents are initialized before their children?  That is why I previously did an `unshift`.  Make sense?